napari/_qt/layer_controls/qt_layer_controls_base.py

```diff
@@ -88,6 +90,8 @@ def changeBlending(self, text):
             Name of blending mode, eg: 'translucent', 'additive', 'opaque'.
         """
         self.layer.blending = self.blendComboBox.currentData()
+        # GL minimum blending does not support changing alpha
+        self.opacitySlider.setVisible(self.layer.blending != 'minimum')
```

Comment:
```suggestion
        self.opacitySlider.setVisible(self.layer.blending != 'minimum')
        self.blendComboBox.toolTip("text with information" if self.layer.blending == 'minimum' else "")
        if self.layer.blending == 'minimum':
        	layer.help = "text with information"
```

The first new line is a tooltip for the mode selector. The third and fourth will show text in the bottom right of the viewer.

Reply:
I've implemented both of these. 
On paper, they're great; in practice, less so.
The `self.layer.help` bit doesn't fire till you mouse the canvas. Took *me* a while to figure that out, so I'm not sure how helpful that is. 
The tooltip is *slow*, you really have to dwell on it. It doesn't fire when a somewhat frustrated user uses the mouse to change blending (like *me* again—it took me way to long to see these work!).
Better than nothing I guess.

Maybe another option is to add an example? (such as the above code in my OP)

Reply:
> The `self.layer.help` bit doesn't fire till you mouse the canvas. Took _me_ a while to figure that out, so I'm not sure how helpful that is.

We may try to set status, which will be put in the left part of the status bar, but this will require passing the viewer to Layer Controls. 

All other things that I have in mind will be more invasive (easier to notice) and will be equivalent to showing a message dialog that, in the meeting, you point out as too invasive.

Reply:
Yea, it is what it is. 
What do you think of providing a simple example?
I'm not sure if people use them much or can discover them easily...

Reply:
I do not think that It will increase discoverability. But maybe add a QLabel, at the end of controls, with short text that will be displayed when a minimum mode is selected?

Reply:
Would have to be super short... 
I'm not a fan, maybe someone else can chime in.